blueocean-dashboard/src/main/js/components/PipelineRowItem.jsx

```diff
@@ -7,14 +7,16 @@ import { capable, UrlConfig } from '@jenkins-cd/blueocean-core-js';
 import { MATRIX_PIPELINE } from '../Capabilities';
 import { Icon } from '@jenkins-cd/react-material-icons';
 
-function generateRedirectLink(pipeline) {
+function generateRedirectLink(pipeline, organization) {
     if (capable(pipeline, MATRIX_PIPELINE)) {
+        const fullDisplayPath = organization ? `${organization}/${pipeline.fullDisplayName}` : pipeline.fullDisplayName;
```

Comment:
... only thing, is this the display name of the organization; seems inconsistent with the `pipeline` object, maybe rename it to `organizationDisplayName` to be clear....